### PR TITLE
Windows/Client Management: Note blob format typo correction

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #8242 , 
Note blob format typo correction

### CODE REVIEW

**Comment by illfated** [comment]
The page URL has recently become available again, with slightly different phrasing, referring to itself being an archived version.
Please check out the download page for Network Monitor, https://www.microsoft.com/en-us/download/details.aspx?id=4865 .
If the page will remain available like this for an extended period of time, we may want to close this pull request.

**Comment by illfated** [comment]
https://www.microsoft.com/en-us/download/details.aspx?id=4865

image

**Comment by illfated** [comment]
# Collect data using Network Monitor

In this topic, you will learn how to use Microsoft Network Monitor 3.4, which is a tool for capturing network traffic.

> [!NOTE]
> Network Monitor is the archived protocol analyzer and is no longer under development. **Microsoft Message Analyzer** is the replacement for Network Monitor. For more details, see [Microsoft Message Analyzer Operating Guide](https://docs.microsoft.com/message-analyzer/microsoft-message-analyzer-operating-guide).

To get started, [download and run NM34_x64.exe](https://www.microsoft.com/download/details.aspx?id=4865). When you install Network Monitor, it installs its driver and hooks it to all the network adapters installed on the device. You can see the same on the adapter properties, as shown in the following image.

Adapters

When the driver gets hooked to the network interface card (NIC) during installation, the NIC is reinitialized, which might cause a brief network glitch.

File: windows/client-management/troubleshoot-tcpip-netmon.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ In this topic, you will learn how to use Microsoft Network Monitor 3.4, which is
 > [Note]
```
Comment (illfated) [inline windows/client-management/troubleshoot-tcpip-netmon.md:19]: There is still a possible use for this PR, although that requires renaming the title to something like
**Windows/Client Management: Note format adjustment**
```suggestion
> [!NOTE]
```

File: windows/client-management/troubleshoot-tcpip-netmon.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ manager: dansimp
 
 In this topic, you will learn how to use Microsoft Network Monitor 3.4, which is a tool for capturing network traffic.
 
-> [Note]
+> [!NOTE]
 > Network Monitor is the archived protocol analyzer and is no longer under development. **Microsoft Message Analyzer** is the replacement for Network Monitor. For more details, see [Microsoft Message Analyzer Operating Guide](https://docs.microsoft.com/message-analyzer/microsoft-message-analyzer-operating-guide).
 
-To get started, [download and run NM34_x64.exe](https://www.microsoft.com/download/details.aspx?id=4865). When you install Network Monitor, it installs its driver and hooks it to all the network adapters installed on the device. You can see the same on the adapter properties, as shown in the following image.
+To get started, **run NM34_x64.exe**(we are leaving this page available for those who have downloaded the tool previously). When you install Network Monitor, it installs its driver and hooks it to all the network adapters installed on the device. You can see the same on the adapter properties, as shown in the following image.
```
Comment (illfated) [inline windows/client-management/troubleshoot-tcpip-netmon.md:22]: Thank you for merging my previous suggestion. Due to the download being available again, I believe we can revert to the original text in this sentence:
```suggestion
To get started, [download and run NM34_x64.exe](https://www.microsoft.com/download/details.aspx?id=4865). When you install Network Monitor, it installs its driver and hooks it to all the network adapters installed on the device. You can see the same on the adapter properties, as shown in the following image.
```

**Comment by illfated** [comment]
Very good. Thank you again for the follow-up.

Would you be able to rename the PR title to something like the following?

**Windows/Client Management: Note format adjustment**

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
Closing because the download link has became available, and the corresponding issue had been closed.

**Comment by illfated** [comment]
@e0i : I would like to have this PR re-opened, because the PR has changed to a format typo correction.
(Please note the change in both the PR title and the source content.)

**Comment by e0i** [comment]
@JohanFreelancer9 Please review and confirm the typo in formatting. Thank you.

**Comment by illfated** [comment]
@RAJU2529 : Could you also edit your first PR comment to use the following text?

Note blob format typo correction.

**Comment by RAJU2529** [comment]
@illfated . this request already closed .

**Comment by illfated** [comment]
Thank you for the feedback, but it has been reopened again. Please notice the green PR symbol (see image below):

image

**Comment by illfated** [comment]
@RAJU2529 : So what I am asking for, is that you edit your first PR comment and remove the following text:

[as per the user report #8242 , so I removed the download link and added an explanation.]

and replace it with a text explaining more precisely what is done here, like for example

[Note blob format typo correction.]

**Review by JohanFreelancer9 — APPROVED** [review]
The NOTE format change is correct.

**Review by illfated — APPROVED** [review]
Looks OK to me.

**Comment by illfated** [comment]
(ref. #8282 )

**Pull request merged**